tests/test_block_cl.py

```diff
@@ -426,7 +426,7 @@ def test(self):
                 "conf_selector": self.conf_selector,
                 "fp_config": {"inputs": self.vasp_inputs},
                 "lmp_task_grp": self.task_group_list,
-                "optional_parameter": {"data_mixed_type": True},
+                "optional_parameter": {"data_mixed_type": True, "do_finetune": "no"},
```

Comment:
A minor point, default value should not be specified explicitly. To achieve this, always get value from this dict with a default value.